public/docs/ts/latest/cli-quickstart.jade

```diff
@@ -122,157 +110,263 @@ figure.image-display
-  that controls the appearance and behavior of a view through its template.
-
-a#component-import
-:marked
-  ### Import
-
-  Angular apps are modular. They consist of many files, each dedicated to a purpose.
-
-  Angular itself is modular. It is a collection of library modules
-  consisting of several, related features that we'll use to build our application.
-
-  When we need something from a module or library, we import it.
-  Here we import the `Component` decorator from the Angular **_core_** library 
-  that we'll use as a decorator (`@Component`) to attach metadata to our component class.
-  .
-
-+makeExcerpt('src/app/cli-quickstart.component.ts', 'import')
-
-h3#component-decorator @Component decorator
-:marked
-  `Component` is a *decorator* function that associates Angular *metadata* with a
-  component class.
-  The metadata tell Angular how to create and display instances of this component.
-
-  We apply this function to the component class by prefixing the function name with the
-  **@** symbol and invoking it with a metadata object, just above the class.
-
-+makeExcerpt('src/app/cli-quickstart.component.ts', 'metadata')
-
-:marked
-  This particular metadata object has four fields, a `moduleId`, a `selector` a `templateUrl` and an array of `styleUrls`.
-  
-  The **moduleId** specifies the location of _this_ component definition file
-  so that Angular can find the corresponding _template_ and _style_ files with URLs that
-  are relative to this component file. 
-  The module loader sets the value of `module.id` at runtime;
-  we're using that value to set the metadata `moduleId` property.
-
-  The **selector** is a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_Started/Selectors) 
-  identifying the HTML element that represents this component.
+  Or stick around a bit longer to learn about the files in your brand new project.
 
-  The element tag name for this component is `cli-quickstart-app`.
-  Angular creates and displays an instance of our `CliQuickstartAppComponent`
-  wherever it encounters a `<cli-quickstart-app>` element in the host HTML.
-
-  The **templateUrl** locates the component's companion template HTML file.
-  The template tells Angular how to render this component's view.
-  
-  Our template is a single `<h1>` element surrounding some peculiar Angular data binding syntax.
-+makeExample('src/app/cli-quickstart.component.html', null, 'src/app/cli-quickstart.component.html')(format='.')
+.l-main-section
 :marked
-  The `{{title}}` is an _interpolation_ binding that causes Angular to display the component's
-  `title` property. After out edit, Angular displays "Hello Angular".
-  We'll learn more about data binding as we read through the documentation.
-  
-  The **styleUrls** array specifies the location(s) of the component's private CSS style file(s).
-  The CLI generated an empty file for us and we added styles to it.
+  ## Popping open the trunk
```

Comment:
It is an expression in English. I fear it's a bit too colloquial for our guide. I know it's pedestrian but how about
```
## Project file review
```

* Cut the marketeering about "rock solid".  Not sure what "enterprise" means anymore either. I think the following will do:
```
An Angular-CLI  project is the  foundation for both quick experiments and enterprise solutions.
```